data/types.go

```diff
@@ -208,11 +244,13 @@ func (f TargetFileMeta) HashAlgorithms() []string {
 }
 
 type Targets struct {
```

Comment:
should this also include a Delegations JSON object? It seems like keys are added into a Keys JSON dictionary, rather than nested in a Delegations object like these examples:
https://github.com/theupdateframework/tuf/blob/f4ffb9dbaabfa6e0df22091b7a6ca63f0dec9377/tests/repository_data/repository/metadata/targets.json#L10-L11